src/decoder/external_decoder.ml

```diff
@@ -100,9 +100,10 @@ let test_kind f filename =
   else
     Some
       {
-        Frame.video = Frame.Zero;
-        midi = Frame.Zero;
-        audio = (if ret < 0 then Frame.Succ Frame.Any else Frame.mul_of_int ret);
+        Frame.video = 0;
+        midi = 0;
+        (* TODO: this is not perfect *)
+        audio = (if ret < 0 then Lazy.force Frame.audio_channels else ret);
```

Comment:
Yeah it's not perfect but it would be implicitly converted via the new decoder workflow.

Reply:
Alright then :)